hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/store/NearCacheObjectRecordStore.java

```diff
@@ -53,7 +52,7 @@ protected long getRecordStorageMemoryCost(NearCacheObjectRecord record) {
     }
 
     @Override
-    protected NearCacheObjectRecord<V> valueToRecord(V value) {
+    protected NearCacheObjectRecord<V> createRecordFromValue(V value) {
```

Comment:
can be only `createRecord`? `createRecordFromValue` implies me as if we are reading some fields of value and creating a new record accordingly.